mailerlite.com.website-subdomain.json

```diff
@@ -0,0 +1,18 @@
+{  
+   "providerId": "mailerlite.com",
+   "providerName": "MailerLite",
+   "serviceId": "website",
+   "serviceName": "MailerLite Sites",
+   "version": 1,
+   "description": "Create websites or landing pages using your own domain.",
+   "syncPubKeyDomain": "mailerlite.com",
+   "variableDescription": "Self explanatory in the variable names.",
+   "records": [
+      {
+         "type": "CNAME",
+         "host": "%subdomain_host%",
```

Comment:
For subdomain template host is typically set to `@` and `hostRequired` set to true.
With the usage of `host` variable the CNAME subdomain is created on the right subdomain.